dags/hivemind_etl_helpers/src/db/gdrive/gdrive_loader.py

```diff
@@ -0,0 +1,42 @@
+from typing import List, Optional
+
+from llama_index.core.schema import Document
+from llama_index.readers.google import GoogleDriveReader
+
+
+class GoogleDriveLoader:
+    def __init__(self, client_config):
+        self.client_config = client_config
+
+    def load_data(
```

Comment:
Suggestion for improvement: 

Please have a look on the codeClimate warning
> Function `load_data` has a Cognitive Complexity of 8 (exceeds 5 allowed). Consider refactoring.

You can either define a function for each for loops or use another idea to reduce the complexity

Reply:
Also, please note that the coverage of this file is 36.84%
Please consider improving it.